### DISCUSSION TITLE
Is there a way to install a specific version / downgrade to a specific patch version?

### DISCUSSION THREAD

**Discussion by alexander-schranz** [shivammathur/homebrew-php#1824] [Q&A / Help] [answered]
I have currently 8.1.21 installed but need for testing downgrade to 8.1.18. Is there a way to achieve this?

**Comment by shivammathur** [comment, marked as answer]
https://stackoverflow.com/questions/75835275/how-to-install-specific-older-php-8-1-version-on-mac-using-brew/75848252#75848252

**Reply by alexander-schranz** [reply]
Thx that looks promising. Do you have a hint maybe also for the following error after it instaleld `8.1.18` in my case I run into following error when run `php -v`:

```
dyld[22531]: Library not loaded: '/opt/homebrew/opt/icu4c/lib/libicuio.72.dylib'
  Referenced from: '/opt/homebrew/Cellar/php@8.1/8.1.18_1/bin/php'
  Reason: tried: '/opt/homebrew/opt/icu4c/lib/libicuio.72.dylib' (no such file), '/usr/local/lib/libicuio.72.dylib' (no such file), '/usr/lib/libicuio.72.dylib' (no such file), '/opt/homebrew/Cellar/icu4c/73.2/lib/libicuio.72.dylib' (no such file), '/usr/local/lib/libicuio.72.dylib' (no such file), '/usr/lib/libicuio.72.dylib' (no such file)
fish: Job 1, 'php --version' terminated by signal SIGABRT (Abort
```

**Update**: Answer see: https://github.com/shivammathur/homebrew-php/discussions/1824#discussioncomment-6477375

**Comment by shivammathur** [comment]
Please Install it from the source, that way you will link it to the latest libraries.
```
brew reinstall -s shivammathur/php/php@8.1
```

**Reply by alexander-schranz** [reply]
Works, Thank you!
